### ISSUE TITLE
MS Teams | Automation API & Accessibility Insight | Dual Monitor

### ISSUE THREAD

**Issue by ashubabu** [MicrosoftDocs/msteams-docs#3196]
I had raised an issue with Accessibility Insight Team  [here](https://github.com/microsoft/accessibility-insights-windows/issues/1114) and on [stackoverflow](https://stackoverflow.com/questions/67189742/ms-teams-accessibility-insight-dual-monitor) . In response to Stackoverflow question, Accessibility Team has mentioned that the issue seems be with MS Teams Application in supporting Accessibility.

### Describe the bug
Accessibility behavior of MS Teams on Dual Monitor, with Monitors setup at different scales, my specific example 100% and 125%, with 1920*1080 resolution (or with different combinations of Scaling, Resolution and Physical dimensions of the Monitors in Dual/Multi Monitor setups) do not seem to follow any deterministic behavior. Consequently, element detection is incorrect. I have tried to check the result from Automation API and Accessibility Insight and they both show similar behavior with unproportionate offsets.

From Task Manager, I see MS Teams Process, responsible for UI, is a Per Monitor DPI Aware and so is Accessibility Insight, I also have developed a prototype using Windows Automation API that runs on Per Monitor DPI aware mode. However, both the Prototype (and Accessibility Insight) are unable to locate the MS Teams' Elements correctly in dual Monitor, different scale configuration. The behavior is non-deterministic and is clearly visible with below settings.

Further, MS Teams responds well on single Monitor with any scale factor or resolution for elements that I have tested. 

### To Reproduce

1. Use Dual Monitors with Monitor 1 as Primary. Monitor 1 is set at 100%, 1920x1080 while Monitor 2 is set at 125%, 1920x1080. Both of Monitors are 14 inches which keeps Physical Screen Size out of equation.
2. Launch Teams App in Monitor 1 (100%) and move it to Monitor 2 (125%)
3. Open Accessibility Insight in Monitor 1 (or Monitor 2)
4. Try to locate "Search" box on Title Bar of Teams, where you can Search for a Person. You can try any other Element as well.
5. The Bounding Rectangle I see has Left as 2935 and Top as 280 while the actual Physical Coordinates that I get using GetPhysicalCursorPos (user32.dll) is X:2454 & Y:10
6. I wrote a Prototype that hosts Windows Automation API (code snippet below) and ran it in DPI Unaware, System DPI Aware and Per Monitor DPI Aware mode. In DPI Unaware Mode and System DPI Aware Mode, I get Bounding Rectangle Left as 2348 and Top as 10. If I scale the Bounding Rectangle Left value, obtained in step 5, down by 1.25 to account for Scaling (2935/1.25), I get 2348. Which makes me believe that scaling is done on presuming the Monitor is on 100%, probably because it was launched on Monitor 1 with 100% scale factor.
7. When I run the Prototype in Per Monitor DPI Aware mode, I get value Bounding Rectangle Left and Top as 2935 and 280, that matches with Accessibility Insight.
8.  I have also tried 14 inch and 22 inch monitors' combinations with 100% and 125% respectively and 1920*1080 resolutions. The result is left seems to be aligned however Top still gives an offset of 280.

### Expected behavior

Considering Teams app runs in Per Monitor DPI Aware mode and both the Monitors are 14 inches, I would expect the Prototype (and Accessibility Insight) to give the correct location of the Element/s. In this case it should be Left ~2454 and Top ~10 (Or if I consider scaling keeping physical dimensions and resolutions same on both the monitors, it should be Left ~2454*1.25 and Top ~10*1.25)

### Actual behavior

Teams App doesn't seem to respond correctly to Monitor Scaling.
Screenshots or .GIF

![image](https://user-images.githubusercontent.com/18587133/116456180-950e7e00-a87f-11eb-9724-a311704c0d7b.png)

### Prototype hosting Automation API 

`        {
            SetProcessDpiAwareness(PROCESS_DPI_AWARENESS.Process_Per_Monitor_DPI_Aware);

            Console.WriteLine("Awareness mode {0}", PROCESS_DPI_AWARENESS.Process_Per_Monitor_DPI_Aware.ToString());
            var processes = Process.GetProcessesByName("Teams").Where(p => p.MainWindowTitle.Trim() != "");
            var process = processes.FirstOrDefault();
            IntPtr handle = process.MainWindowHandle;
            _currentHandle = handle;

            _automationRootElement = _automation.ElementFromHandle(_currentHandle);
            var br = _automationRootElement.CurrentBoundingRectangle;
            Console.WriteLine("Root element's position left:{0} top:{1} right:{2} bottom:{3}", br.left, br.top, br.right, br.bottom);
            _walkerx = _automation.ControlViewWalker;

            var cp = new CursorPoint();
            GetPhysicalCursorPos(ref cp);
            Console.WriteLine("Physical Cursor Position X:{0} Y:{1}", cp.X, cp.Y);

            Console.WriteLine("***Automation API***");
            var conditions = new IUIAutomationCondition[1];
            conditions[0] = _automation.CreatePropertyCondition(UIA_PropertyIds.UIA_ControlTypePropertyId, 50004);

            var element = _automationRootElement.FindFirst(TreeScope.TreeScope_Descendants, _automation.CreateAndConditionFromArray(conditions));
            Console.WriteLine("Found element {0}", element.CurrentName);
            Console.WriteLine("Automation API Left: {0}, Top: {1}", element.CurrentBoundingRectangle.left, element.CurrentBoundingRectangle.top);

            Console.ReadLine();
        }`

### System Details

OS: [e.g. Windows 10 1909]
Accessibility Insights for Windows Version:
Target Application: [MS Teams]
Target Application Version: [e.g. 1.4.00.8872]

### Additional context
If I enable GDI scaling, reading [Improve High DPIExperience](https://blogs.windows.com/windowsdeveloper/2017/05/19/improving-high-dpi-experience-gdi-based-desktop-apps/), I do see that Prototype as well as Accessibility Insight is able to locate the Element as it should. Further, Accessibility Insight works well on "Display Settings" itself (SystemSettings.exe process), which is also Per Monitor Aware.
